plugins/netskope/icon_netskope/actions/apply_pending_url_list_changes/schema.py

```diff
@@ -0,0 +1,172 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Applies currently pending changes for URL lists"
+
+
+class Input:
+    pass
+
+class Output:
+    DEPLOYED_URLLISTS = "deployed_urllists"
+    
+
+class ApplyPendingUrlListChangesInput(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {}
+    """)
+
+    def __init__(self):
+        super(self.__class__, self).__init__(self.schema)
+
+
+class ApplyPendingUrlListChangesOutput(insightconnect_plugin_runtime.Output):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "deployed_urllists": {
+      "type": "array",
+      "title": "Deployed URL Lists",
+      "description": "Contains list of deployed URL lists",
+      "items": {
+        "$ref": "#/definitions/urllist"
+      },
+      "order": 1
+    }
+  },
+  "required": [
+    "deployed_urllists"
+  ],
+  "definitions": {
+    "data": {
+      "type": "object",
+      "title": "data",
+      "properties": {
+        "json_version": {
+          "type": "integer",
+          "title": "JSON version",
+          "description": "Current version of JSON format",
+          "order": 3
+        },
+        "type": {
+          "type": "string",
+          "title": "URL List Category",
+          "description": "Category of URL list",
+          "order": 2
+        },
+        "urls": {
+          "type": "array",
+          "title": "URLs",
+          "description": "List of URLs",
+          "items": {
+            "type": "string"
+          },
+          "order": 1
+        }
+      },
+      "required": [
+        "type",
+        "urls"
+      ]
+    },
+    "urllist": {
+      "type": "object",
+      "title": "urllist",
+      "properties": {
+        "data": {
+          "$ref": "#/definitions/data",
+          "title": "Data",
+          "description": "Data containing URLs and type",
+          "order": 3
+        },
+        "id": {
+          "type": "integer",
+          "title": "ID",
+          "description": "URL list identifier",
+          "order": 1
+        },
+        "modify_by": {
+          "type": "string",
+          "title": "Modify By",
+          "description": "Name of the URL list modifier",
+          "order": 5
+        },
+        "modify_time": {
```

Comment:
If it is a datetime why are you using String instead of datetime

Reply:
Well we've talking about that yesterday and right know we don't know exactly if it's better to use `string` or `date` type to that output. What do you think about that?